rasa/core/events/__init__.py

```diff
@@ -83,7 +83,7 @@ def md_format_message(
 
 def first_key(d: Dict[Text, Any], default_key: Any) -> Any:
     if len(d) > 1:
-        for k, v in d.items():
+        for k in d.items():
```

Comment:
In my opinion it's more confusing to only use the first value from a method that returns two values than to have an unused variable. what about this instead? 
```suggestion
        for k in d.keys():
```